### PR TITLE
MNT: more mds api issues

### PR DESCRIPTION
**Pull request by ericdill**
- also change so that live data can be viewed

### CODE REVIEW

**Comment by danielballan** [comment]
Good. Is this ready to merge? Concurrently, I am adding tests that will cover this.

**Comment by ericdill** [comment]
Yes this is ready to merge

File: dataportal/broker/simple_broker.py [hunk 1]
```diff
@@ -195,13 +196,14 @@ def _build_header(run_start):
...
     adds = {'start_time': (run_start, 'time'),
-                'start_datetime': (run_start, 'time_as_datetime')}
+            'start_datetime': (run_start, 'time_as_datetime')}
     deletes = [(run_start, '_name')]
     add_to_id = {'run_start_uid': (run_start, 'uid'),
                  'run_start_id': (run_start, 'id')}
     if run_stop is not None:
         adds['stop_time'] = (run_stop, 'time')
         adds['stop_datetime'] = (run_stop, 'time_as_datetime')
+        adds['exit_report'] = (run_stop, 'reason')
```
Comment (danielballan) [inline dataportal/broker/simple_broker.py:206]: Wait, why "exit_report"?

**Comment by stuwilkins** [comment]
I thought that reason was going to be something simple like a ENUM to say if the scan is good or if it blew up?

**Comment by ericdill** [comment]
In the mds docs for the `RunStop` document, there are two keys called `reason` and `completion_state`.  `completion_state` is the enum with options of `success`, `abort`, `fail`.  

In the metadatastore implementation (https://github.com/NSLS-II/metadatastore/blob/master/metadatastore/odm_templates.py#L67) `reason` is still called `reason` and is a string field which can have any bits of information in it.  `completion_state` is called `exit_status` and has the same enum options.  

The run header that @danielballan  commented on is a conglomeration of all non-data documents from metadatastore.  At the suggestion of @danielballan (and one that I agree with), the contents of the `RunStart` and `RunStop` documents should be dumped into the top level namespace of the `Header` object.  To do this, some name collisions occur (such as RunStart.time and RunStop.time).  

Today, Claudio pointed out that when we do this, the `reason` key from the `RunStop` document becomes `Header.reason` which is somewhat misleading as one would assume that `Header.reason` is the reason that the scan was performed.  To alleviate this, he suggested (and it seemed pretty reasonable to me) that we could remap `RunStop.reason` to `Header.exit_report` so that it is more consistent with the existing enum field: `RunStop.exit_status` that gets dumped into the header resulting in `Header.exit_status`.

Hopefully this brain dump helps...

**Pull request merged**